features/admin.policy-administration.v1/components/policy-administration-page-layout.tsx

```diff
@@ -184,12 +169,80 @@ const PolicyAdministrationPageLayout: FunctionComponent<PolicyAdministrationPage
         setPageInactive((prevPage: number) => prevPage + 1);
     };
 
-
     const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
         event.preventDefault();
+        setPageActive(0);
+        setPageInactive(0);
+        activeScrollRef.current.scrollTo({ behavior: "smooth", top: 0 });
+        inactiveScrollRef.current.scrollTo({ behavior: "smooth", top: 0 });
         setSubmittedSearchQuery(searchQuery);
     };
 
+    /**
+     * Handle deletion of a policy from the active policy list.
+     *
+     * @param policyId - The policy ID.
+     */
+    const handleDeleteActivePolicy = (policyId: string) => {
+        const newActivePolicies: Map<string, PolicyInterface> = new Map(activePolicies);
+
+        newActivePolicies.delete(policyId);
+        setActivePolicies(newActivePolicies);
+        setPageActive(0);
+        mutateActivePolicies();
+    };
+
+    /**
+     * Handle deletion of a policy from the inactive policy list.
+     *
+     * @param policyId - The policy ID.
+     */
+    const handleDeleteInactivePolicy = (policyId: string) => {
+        const newInactivePolicies: Map<string, PolicyInterface> = new Map(inactivePolicies);
+
+        newInactivePolicies.delete(policyId);
+        setInactivePolicies(newInactivePolicies);
+        setPageInactive(0);
+        mutateInactivePolicies();
```

Comment:
Does L203 really delete the policy from server?

Reply:
Ack. The function name implied it was triggering a permanent delete operation though, that's why i wondered :)